test/memory.test.js

```diff
@@ -281,21 +280,27 @@ describe('Memory connector', function() {
     });
 
     it('should successfully extract 2 users using implied and', function(done) {
-      User.find({ where: { role:'lead', vip:true }}, function(err, users) {
+      User.find({ where: { role: 'lead', vip: true }}, function(err, users) {
         should(users.length).be.equal(2);
         should(users[0].name).be.equal('John Lennon');
         should(users[1].name).be.equal('Paul McCartney');
         done();
       });
     });
 
-    it('should successfully extract 2 users using implied and & and', function(done) {
-      User.find({ where: { name: 'John Lennon', and: [{ role:'lead' }, { vip:true }] }}, function(err, users) {
-        should(users.length).be.equal(1);
-        should(users[0].name).be.equal('John Lennon');
-        done();
+    it('should successfully extract 2 users using implied and & and',
+      function(done) {
```

Comment:
To prevent whitespace changes when fixing lines too long, we usually don't indent the callback:

```js
it('should successfully extract 2 users using implied and & and',
function(done) {
  // indentation stays the same here
});
```